src/main/java/org/opensearch/knn/index/IndexUtil.java

```diff
@@ -321,4 +358,26 @@ public static boolean isBinaryIndex(KNNEngine knnEngine, Map<String, Object> par
             && parameters.get(VECTOR_DATA_TYPE_FIELD) != null
             && parameters.get(VECTOR_DATA_TYPE_FIELD).toString().equals(VectorDataType.BINARY.getValue());
     }
+
+    /**
+     * Tell if it is binary index or not
+     *
+     * @param vectorDataType vector data type
+     * @return true if it is binary index
+     */
+    public static boolean isBinaryIndex(VectorDataType vectorDataType) {
```

Comment:
Move all public functions above private functions in this class.

Reply:
done